src/main/java/examples/MetricsExamples.java

```diff
@@ -30,10 +30,12 @@
 public class MetricsExamples {
```

Comment:
I guess this should be deleted and any metrics examples should be added to the vertx-examples repo instead